Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/WANDPowderReduction.py

```diff
@@ -0,0 +1,129 @@
+from __future__ import absolute_import, division, print_function
+from mantid.api import (DataProcessorAlgorithm, AlgorithmFactory,
+                        MatrixWorkspaceProperty, PropertyMode)
+from mantid.simpleapi import (ConvertSpectrumAxis, Transpose,
+                              ResampleX, CopyInstrumentParameters,
+                              Divide, DeleteWorkspace, Scale,
+                              MaskAngle, ExtractMask, Minus,
+                              RemoveMaskedSpectra, mtd)
+from mantid.kernel import StringListValidator, Direction, Property
+
+
+class WANDPowderReduction(DataProcessorAlgorithm):
+    temp_workspace_list = ['__data_tmp', '__cal_tmp', '__mask_tmp', '__bkg_tmp']
+
+    def category(self):
+        return "Diffraction\\Reduction"
```

Comment:
[Python raw string](https://docs.python.org/2.0/ref/strings.html) would work as well. Doesn't need to be changed unless you really want to.

Reply:
This is consistent with every other algorithm.